package-inspector/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
+    defaultConfig {
+        applicationId "com.microsoft.shatestapp"
+        minSdkVersion 19
```

Comment:
suggest using version.properties for all the values here - so that we can verify that bumping SDK version doesn't break the scenario.  (i.e. rootProject.ext.minSdkVersion)

Reply:
Good idea, thanks @rpdome

Reply:
Fixed in bd2adbe630782e602c5cf519e43f35dd34d7a8af